.gitignore

```diff
@@ -23,3 +23,7 @@ build/Release
 # Deployed apps should consider commenting this line out:
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
+
+# Ignore Visual Studio files
+.sln
+.njsproj
```

Comment:
POSIX standard demands that every file is finished by a newline character:

https://robots.thoughtbot.com/no-newline-at-end-of-file

_A source file that is not empty shall end in a new-line character, which shall not be immediately preceded by a backslash character._